docs/en/compute/Compute-Hive.md

```diff
@@ -182,13 +193,81 @@ Instructions and examples till here illustrate how to use Alluxio as one of the
 tables in Hive, together with other filesystems like HDFS. They do not require to change the global
 setting in Hive such as the default filesystem which is covered in the next section.
 
-## Use Alluxio as the Default Filesystem
+## Advanced Setup
+
+### Add Additional Alluxio client-side properties
+
+There are two ways to specify any Alluxio client properties for Hive queries when
+connecting to Alluxio service:
+
+- Specify the Alluxio client properties in `alluxio-site.properties` and
+ensure that this file is on the classpath of Hive service on each node.
+- Add the Alluxio site properties to `conf/hive-site.xml` configuration file on each node.
+
+For example, change
+`alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`.
+
+One can specify the property in `alluxio-site.properties` and distribute this file to the classpath
+of each Hive node:
+
+```properties
+alluxio.user.file.writetype.default=CACHE_THROUGH
+```
+
+Alternatively, modify `conf/hive-site.xml` to have:
+
+```xml
+<property>
+  <name>alluxio.user.file.writetype.default</name>
+  <value>CACHE_THROUGH</value>
+</property>
+```
+
+You can use `-h` to display helpful information about the command.
```

Comment:
This is related to the `alluxio-checker.sh` and should be put in the end of this file

Reply:
oops. good catch